### DISCUSSION TITLE
how should a multi-employer healthcare plan be represented?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#539] [Q&A] [answered]
What should be used for the plan_id for a [multi-employer plan](https://www.ifebp.org/news/featuredtopics/multiemployer/Pages/default.aspx)  in the files since the plan doesn't have a HIOS or a single EIN?

The EIN could be the EIN of the carrier, but then we are relying only on the plan name for identification.

**Comment by dougcottrell** [comment]
We have the same question.

**Comment by Devyandu** [comment, marked as answer]
In this scenario we would have the same plan_id (EIN of carrier) and different plan names.
There is nothing in the regulation or schema that says that the plan_id needs to be unique.

In a[ prior post](https://github.com/CMSgov/price-transparency-guide/discussions/44#discussioncomment-2595416) (for a different context) @shaselton-usds  said.

> plan_name + plan_id_type + plan_id + plan_market_type is the composite key for uniqueness in plan reporting

**Reply by nmayle** [reply]
Right. On that issue, we see a problem in a table-of-contents/index file. 

The index file has more than one reporting structure objects and each reporting structure has one plan with a composite key (plan_name_a + plan_id_type_a + plan_id_a + plan_market_type_a ), but has two different allowed_amount_files, so that they are associating two different allowed_amounts_files with the same plan a.

As follows:
```
{
 "reporting_plans": [
        {
          "plan_name": "plan-a",
          "plan_id_type": "ein",
          "plan_id": "plan-a-ein",
          "plan_market_type": "group"
        }
   ],
 "allowed_amount_file": {
        "description": "allowed amount file",
        "location": "location-url-1"
   }
 },
{
 "reporting_plans": [
        {
          "plan_name": "plan-a",
          "plan_id_type": "ein",
          "plan_id": "plan-a-ein",
          "plan_market_type": "group"
        }
   ],
 "allowed_amount_file": {
        "description": "allowed amount file 2",
        "location": "location-url-2"
   }
 },
}
```

I assume this is not valid and should be corrected by the creator of the files. Correct?

Also, I would imagine this is something the validator is not designed to detect.
